python/src/FunctionalChaosRandomVector_doc.i.in

```diff
@@ -29,34 +29,34 @@ functionalChaosResult : :class:`~openturns.FunctionalChaosResult`
 %feature("docstring") OT::FunctionalChaosRandomVector::getMean
 "Accessor to the mean of the functional chaos expansion.
 
-Let :math:`n_X \in \Nset` be the dimension of the input random vector, 
-let :math:`n_Y \in \Nset` be the dimension of the output random vector.
+Let :math:`\physicalInputDimension \in \Nset` be the dimension of the input random vector, 
+let :math:`\physicalOutputDimension \in \Nset` be the dimension of the output random vector.
 and let :math:`P \in \Nset` be the number of coefficients in the basis.
 We consider the following functional chaos expansion:
 
 .. math::
 
-      \widetilde{\vect{Y}} = \sum_{k = 0}^P \vect{a}_k \psi_k(\vect{Z})
+      \widetilde{\physicalOutputRandomVector} = \sum_{k = 0}^P \vect{a}_k \psi_k(\vect{Z})
```

Comment:
The size of the basis is, indeed, P+1. There are two reasons for this. The first one is that the notation is classical in this context. See for example [^LeMaitre2010Spectral] page 34:

![image](https://github.com/openturns/openturns/assets/31351465/765ec287-2d0b-4855-a57d-6aa37bb87b5e)

The second one is that the first coefficient, $a_0$, has a special meaning because it is the expected value of the PCE. Hence some equations involve sums from 1 to P.

[^LeMaitre2010Spectral]: Le Maître, O., & Knio, O. M. (2010). _Spectral methods for uncertainty quantification: with applications to computational fluid dynamics_. Springer Science & Business Media.

Reply:
Then you need to change an earlier sentence (at line 34) which currently states: "let :math:P \in \Nset be the size of the basis."

Reply:
Oups, you are right.

Reply:
OK, thanks.